src/Wrap.js

```diff
@@ -34,28 +35,28 @@ const Wrap = (props: WrapProps): React.Element<*> => {
 
         <linearGradient id={idGradient}>
           <stop offset="0%" stopColor={props.primaryColor}>
-            <animate
+            { showAnimation && <animate
```

Comment:
It is safer to make something like this:
`{ props.animation ? <div /> : null }`

Because it can return `false`, you know?